packages/slate/src/models/point.js

```diff
@@ -355,7 +355,23 @@ class Point extends Record(DEFAULTS) {
     }
 
     const { key, offset, path } = this
-    const target = node.getNode(key || path)
```

Comment:
Can you switch this section back, but maybe use `path || key` instead if that's the ideal case? I'd rather separate out performance improvements into another PR from this one, instead of bundling them in with the changes.

Reply:
Hi Ian,

I tried reverting the changes to the `point` model, as well as just using `path || key`, but I was running into failing tests. See my more complete comments on the changes I made.